guides/rest_doc.md

```diff
@@ -741,6 +745,150 @@ kintoneUtility.rest.getFormLayout({
 });
 ```
 
+## <a name="postDeployAppSettings"> kintoneUtility.rest.postDeployAppSettings(param)
+* Deploy the apps to the production environments.
+
+#### Parameter
+
+| Name | Data type | Required | Description
+|:-----------|:------------:|:------------:|:------------
+| param | Object | Yes |  |
+| param.apps | Object | Yes | The array of the app information.
+| param.apps[].app | Number | Yes | The app ID.
+| param.apps[].revision | Number | Optional | The expected revision number. If the value does not match, an error will occur and the record will not be updated. If the value is not specified or is -1, the revision number will not be checked.
+| param.revert | Boolean | Optional | **Default:** `false`. Set `true` if you want to revert the deploy when some error occur during it.
+| param.isPreview | Boolean | Optional | **Default:** `false`. Set `true` if you want to get pre-live setting.
+| param.isGuest | Boolean | Optional | **Default:** `false`. Set `true` if the app belongs to a guest space.
+
+#### Response
+```
+Promise Object
+```
+#### Sample
+```js
+kintoneUtility.rest.postDeployAppSettings({
+  apps: [
+    { app: 542 },
+  ],
+}).then(function(response) {
+  console.log(response);
+}).catch(function(error) {
+  console.log(error);
+});
+```
+
+## <a name="getAppDeployStatus"> kintoneUtility.rest.getAppDeployStatus(param)
+* Get the status of the deployments of the apps.
+
+#### Parameter
+
+| Name | Data type | Required | Description
+|:-----------|:------------:|:------------:|:------------
+| param | Object | Yes |  |
+| param.apps | Number[] | Yes | The array of the ids of the apps.
+| param.isGuest | Boolean | Optional | **Default:** `false`. Set `true` if the app belongs to a guest space.
+
+#### Response
+```
+Promise Object
+```
+#### Sample
+```js
+kintoneUtility.rest.getAppDeployStatus({
+  apps: [ 542 ],
+}).then(function(response) {
+  console.log(response);
+}).catch(function(error) {
+  console.log(error);
+});
+```
+
+## <a name="getCustomization"> kintoneUtility.rest.getCustomization(param)
+* Get the information of JavaScrit/CSS files used to customize an app.
```

Comment:
Please be noticed that a typo mistake has taken in this description. I recommend to change to "Gets the JavaScript and CSS Customization settings of an App." for fitting API's description.